### ISSUE TITLE
Clicking on Facebook group doesn't respond during initial SMS scan.

### ISSUE THREAD

**Issue by yogev1978** [ScDor/alspam#66]
Clicking on Facebook group doesn't respond during initial SMS scan. When the scan finishes, it finally opens the Facebook group if you clicked on it during the scan.
![Screenshot_20201011-012224_אל-ספאם.jpg](https://user-images.githubusercontent.com/10349199/95666224-435f0a00-0b60-11eb-99ac-6f9c714aea8d.jpg)

**Comment by ScDor** [comment]
fixed on 6bff111cf9e77decaf9a28d4bc7a9a0fd9a41163 (seems to have solved it) and its successor 5100b0cadd0781e2685bd40ed1af62f1465fe472 (a couple of minutes later, just moved the "showmainmenu()" to the UI thread, even though it worked well under the separate thread set in 6bff111cf9e77decaf9a28d4bc7a9a0fd9a41163

**Comment by yogev1978** [comment]
Still doesn't work on my Samsung Galaxy S8. Log sent.

**Comment by yogev1978** [comment]
@roikohno Did you do something on this? Seems to get fixed in version 0.030

**Comment by roikohno** [comment]
Did nothing. Strange. @ScDor ?

**Comment by ScDor** [comment]
Nope. Strange indeed. @yogev1978 did you notice any difference between a clean/dirty installations?
